remix.init/index.js

```diff
@@ -39,24 +33,19 @@ async function copyTemplateFiles({ files, rootDirectory }) {
 
 async function updatePackageJsonForEdge(directory) {
   const packageJson = await PackageJson.load(directory);
-  const {
-    dependencies: { "@remix-run/node": _node, ...dependencies },
-    scripts,
-    ...restOfPackageJson
-  } = packageJson.content;
+  const { dependencies, scripts, ...restOfPackageJson } = packageJson.content;
 
   packageJson.update({
-    // dev script is the same as the start script for Netlify Edge, "cross-env NODE_ENV=production netlify dev"
-    scripts: {
-      ...scripts,
-      dev: 'remix dev --manual -c "ntl dev --framework=#static"',
-    },
     ...restOfPackageJson,
     dependencies: {
       ...dependencies,
       "@netlify/edge-functions": "^2.0.0",
       "@netlify/remix-edge-adapter": "^3.0.0",
-      "@netlify/remix-runtime": "^2.0.0",
```

Comment:
`@netlify/remix-runtime` is a dependency of `@netlify-remix-edge-adapter`

Reply:
I've added it as a direct dependency because we're importing from it inside the site, and package managers with strict dependency handling will complain if you import from transitive dependencies